client/app/src/main/java/com/example/protego/DoctorViewPatientNotes.java

```diff
@@ -6,46 +6,91 @@
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class DoctorViewPatientNotes extends AppCompatActivity {
 
     private Button button;
-    public static ArrayList<PatientNotesActivity.NotesInfo> notesData = PatientNotesActivity.notesData;
-
+    public static List<PatientNotesActivity.NotesInfo> notesData;
+    public static final String TAG = "DoctorViewPatientNotes";
+    private TextView tvFullName;
+    private String pid;
+    private String patientName;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_doctor_view_patient_notes);
-        RecyclerView recyclerView = findViewById(R.id.doctorViewPatientNotesRecyclerView);
-
-        setUpPatientNotes();
 
+        //setUpPatientNotes();
         connectButtonToActivity(R.id.returnFromNotes, DoctorViewPatientSelections.class);
+        DoctorViewPatientNotes thisObj = this;
+        Bundle extras = getIntent().getExtras();
+        patientName = extras.getString("patientFirst");
+        pid = extras.getString("patientId");
+        //System.out.println("Passing through " + pid);
 
+        notesData = new ArrayList<>();
 
-        PatientNotesRecyclerViewAdapter adapter = new PatientNotesRecyclerViewAdapter(this,notesData);
-        recyclerView.setAdapter(adapter);
-        recyclerView.setLayoutManager(new LinearLayoutManager(this));
-
-
-    }
+        FirestoreAPI.getInstance().getNotes(pid, new FirestoreListener<List<Note>>() {
+            @Override
+            public void getResult(List<Note> nList) {
+                String note_id;
+                String title;
+                String date;
+                String visibility;
+                String details;
+
+                System.out.println("Notes List : " + nList);
+
+                RecyclerView rvNotesForDoctors = findViewById(R.id.doctorViewPatientNotesRecyclerView);
+
+                tvFullName = findViewById(R.id.notesPatientFullNameInput);
+                tvFullName.setText(extras.getString("patientFirst"));
+
+                for(Note note : nList){
+                    note_id = note.getNoteID();
+                    title = note.getTitle();
+                    date = note.getDateCreated().toString();
+                    visibility = note.getVisibility();
+                    details = note.getContent();
+
+                    notesData.add(new PatientNotesActivity.NotesInfo(note_id, title, date, visibility, details));
+                    Log.d(TAG, "info : " + note.getTitle());
+                    Log.d(TAG, "info : " + note.getDateCreated().toString());
+                    Log.d(TAG, "info : " + note.getVisibility());
+                    Log.d(TAG, "info : " + note.getContent());
+                }
+
+                // create adapter
+                final PatientNotesRecyclerViewAdapter adapter = new PatientNotesRecyclerViewAdapter(thisObj, notesData);
```

Comment:
Using the same notes view as patients allows a doctor to see the edit and delete buttons for notes. Clicking the delete button results in an error since the deletion is attempted using the doctor's UID. Doctors shouldn't have access to these buttons as a result.